backend/attendance-service/src/main/java/org/egov/validator/StaffServiceValidator.java

```diff
@@ -143,20 +126,28 @@ public void validateStaffPermissionRequestParameters(StaffPermissionRequest staf
         for (String id : uniqueIds) {
```

Comment:
Duplicate can be checked using Set Java API

Reply:
Logic reimplemented using Set